src/main/fc/fc_msp.c

```diff
@@ -534,18 +534,6 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
         }
         break;
 #ifdef USE_PROGRAMMING_FRAMEWORK
-    case MSP2_INAV_LOGIC_CONDITIONS:
```

Comment:
@Scavanger this is a no-go. We can not change the current MSP frame to have a completely different structure. 
Instead of changing `MSP2_INAV_LOGIC_CONDITIONS` let's do the following:
1. Leave `MSP2_INAV_LOGIC_CONDITIONS` unchanged
2. Add `MSP2_INAV_LOGIC_CONDITIONS_SINGLE` frame
3. Switch configurator to use `MSP2_INAV_LOGIC_CONDITIONS_SINGLE`